src/main/java/org/jenkinsci/plugins/JiraTestResultReporter/JobConfigMapping.java

```diff
@@ -57,14 +58,15 @@
          * @param configs list with the configured fields
          */
         public JobConfigEntry(String projectKey, Long issueType, List<AbstractFields> configs,
-                              boolean autoRaiseIssue, boolean autoResolveIssue, boolean preventDuplicateIssue) {
+                              boolean autoRaiseIssue, boolean autoResolveIssue, boolean preventDuplicateIssue,String maxNoofBugs) {
             this.projectKey = projectKey;
             this.issueType = issueType;
             this.configs = configs;
             this.issueKeyPattern = Pattern.compile(projectKey + "-\\d+");
             this.autoRaiseIssue = autoRaiseIssue;
             this.autoResolveIssue = autoResolveIssue;
             this.preventDuplicateIssue = preventDuplicateIssue;
+            this.maxNoofBugs= maxNoofBugs;
```

Comment:
Must this be a String type? Can we use an int? This applies across the entire set of changes. Try to use the appropriate type where appropriate.